css/style.css

```diff
@@ -0,0 +1,232 @@
+html { 
+  background: url("../images/Skola.jpg") no-repeat center center fixed;
+  background-size: cover;
+}
+
+body {
+  margin: 0;
+}
+
+.page-container{
+  position: relative;
+  min-height: 100vh;
+}
+
+.content-wrap{
+  padding-bottom: 15px;
+}
+
+ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  text-align: end;
+  padding-top: 10px;
+  padding-right: 10px;
+}
+
+li {
+  display: inline;
```

Comment:
Do you need this?

Reply:
Yes, they align the links in my navbar horizonally.